src/css/mapbox-gl.css

```diff
@@ -85,8 +85,8 @@
 }
```

Comment:
Since `postcss-nested` is not valid syntax for CSS, I suggest to change extension to `.scss` to save code highlighting in code editors.

In future using `postcss-import` you can [simplify this CSS like this](https://github.com/bravecow/mapbox-gl-controls/blob/b1c1a51fff9c963cf4fcab83ac44788a90266b45/src/theme.scss#L1-L7).  😉